### PR TITLE
Adding TestSets for RigidLinks, Loadcase and Loadcombinations

### LINKED ISSUE #356: Add testsets for RigidLinks
#### Description:

Dataset containing testdata for rigid links to be added

### LINKED ISSUE #358: Add testSets for Cases and Combinations
#### Description:

As description states, add testsets for cases and combinations

### PR DESCRIPTION
**Pull request by IsakNaslundBh**
### Issues addressed by this PR

 Closes #356
Closes #358

 

Adding testsets for rigid links

 ### Test files

 ### Changelog

 ### Additional comments

### CODE REVIEW

**Comment by IsakNaslundBh** [comment]
@BHoMBot check dataset-compliance

**Review by JosefTaylor — APPROVED** [review]
everything seems reasonable, rigidLinks appear named consistent to their constraint. I note that the load combos and loadcases have funny names - all start with 'Name-' followed by something descriptive.

File: Datasets/TestSets/Structure/Loadcase/01 - Test All Natures.json [hunk 1]
```diff
@@ -0,0 +1 @@
... : "Name-Temperature", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Temperature", "Number" : 6 }, { "_t" : "BH.oM.Structure.Loads.Loadcase", "BHoM_Guid" : "639e1fb8-fcde-44bf-b7c6-a8bacf7b9cbf", "Name" : "Name-Snow", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Snow", "Number" : 7 }, { "_t" : "BH.oM.Structure.Loads.Loadcase", "BHoM_Guid" : "11263943-5fc3-4df7-864c-675cef173bf9", "Name" : "Name-Accidental", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Accidental", "Number" : 8 }, { "_t" : "BH.oM.Structure.Loads.Loadcase", "BHoM_Guid" : "acc89e11-158c-4e27-859b-b456e8b6c347", "Name" : "Name-Prestress", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Prestress", "Number" : 9 }, { "_t" : "BH.oM.Structure.Loads.Loadcase", "BHoM_Guid" : "0ed2fb2f-aee8-4ddd-afd7-3708a8c39506", "Name" : "Name-Notional", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Notional", "Number" : 10 }, { "_t" : "BH.oM.Structure.Loads.Loadcase", "BHoM_Guid" : "2ebdc59a-6a52-4b4a-a691-c3f9fbe3331d", "Name" : "Name-Other", "Fragments" : [], "Tags" : [], "CustomData" : { }, "Nature" : "Other", "Number" : 11 }], "_bhomVersion" : "4.3" }
```
[the remark this answers is not in the document]
Comment (IsakNaslundBh) [reply Datasets/TestSets/Structure/Loadcase/01 - Test All Natures.json:1]: It was just to put something there really, and something that could be relatively easily identifiable, without being exactly the same as the nature

**Comment by IsakNaslundBh** [comment]
@BHoMBot check null-handling

**Pull request merged**